### PR TITLE
Book library in JavaScript

### PR DESCRIPTION
**Pull request by NiiazalyDzhumaliev**
On our project, we implemented the following features:

- write a function that loops through the array and displays each book on the page;
- add a “NEW BOOK” button that brings up a form allowing users to input the details for the new book;
- add a button on each book’s display to remove the book from the library;
- add a button on each book’s display to change its read status.

Please kindly review our code.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @NiiazalyDzhumaliev and @paulzay 
### You have done well so far :clap: However, you need to fix a few issues first:-
- [ ] Creates an array to store the instances of a `Book`, [see more details](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png)
- [ ] Solve the errors returned in the console
- [ ] Use es6 function to implement your features, see below
```
const showBooks = () => {
    # codebase
};
```
### Make these changes and submit for another code review. Good luck.
- Happy coding!

File: script.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+};
+const button = document.getElementById('button');
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+  this.info = () => `${this.title} written by ${this.author}, ${this.pages} ${this.read ? 'read' : 'unread'}`;
+}
```
Comment (rOluochKe) [inline script.js]: ### Implements the function using es6

File: script.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    deleteButton.classList.add('delete-btn');
+
+    li.appendChild(changeButton);
+    li.appendChild(deleteButton);
+
+    changeButton.onclick = (self) => {
+      li = self.target.parentElement;
+      changeReadStatus(li.book);
+      showBooks();
+    };
```
Comment (rOluochKe) [inline script.js]: ### Implements the function using es6

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**

**Pull request merged**